content/enable-wayland.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Enabling Wayland
+description: >
+ How to enable Wayland on Pop!_OS
+keywords:
+ - Wayland
+ - Display Manager
+
+hidden: false
+section: software
+tableOfContents: true
+---
+
+**Note:** *Using Wayland is not supported on Pop!\_OS and may cause problems particularly when using Nvidia graphics cards. Do so at your own risk.*
+
+Edit `/etc/gdm3/custom.conf` and comment out the line `WaylandEnable=false`. Save the file and restart your computer.
```

Comment:
This doesn't explain what "commenting out" means, or how to edit this file with elevated privileges. These instructions are not sufficient.